Mathlib/Geometry/Synthetic/Avigad/Axioms.lean

```diff
@@ -0,0 +1,237 @@
+/-
+Copyright (c) 2023 André Hernandez-Espiet. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: André Hernandez-Espiet
+-/
+import Mathlib.Data.Real.Basic
+import Mathlib.Data.Set.Finite
+
+/-!
+Axioms of synthetic geometry
+-/
+
+/-! Universes for points lines and circles-/
+universe u
+
+/-- `IncidenceGeometry` represents geometry in the Euclidean sense, with primitives for points
+lines and circles-/
+class IncidenceGeometry where
```

Comment:
Everything after this `where` should be indented by two spaces

Reply:
Ok, will do

Reply:
I see 4 spaces, not 2.